Pod/Classes/SEGAmplitudeIntegration.m

```diff
@@ -42,7 +42,6 @@ + (NSNumber *)extractRevenueOrTotal:(NSDictionary *)dictionary withRevenueKey:(N
 
         if ([key caseInsensitiveCompare:totalKey] == NSOrderedSame) {
             revenueOrTotal = dictionary[key];
-            break;
```

Comment:
Hmmm... why did this change?

Reply:
We want revenue to be used in cases where both are present

Reply:
You'll want to add this reason as a comment in the codebase so the next developer (e.g. future you) does not look at it and think there's a missed `break` and thus a bug then put it back in.

Reply:
I will say that if a break was added, tests would fail